README.md

```diff
@@ -1,2 +1,71 @@
-# minecraft-talking-blocks-mod
-Experimental TTS blocks for Minecraft that makes things talk!
+# Minecraft Talking Blocks Mod
+
+## OwO!? What's this!? TTS in LEGIT Minecraft!? 
+Don't call us crazy! The Minecraft Talking Blocks Mod is a showcase of our mere attempt into making functional text to speech (TTS) a reality in Minecraft with a rather simple yet pretty old and somewhat arcane Java TTS library. Although very combersome at first and probably by far something that no other Minecraft modder has even dared in trying to do, we might as well be the first in doing so anyways! Sometimes those lines of text going by in your chats with a command block, chat box or whatever other chat things you might as well use just get a little too boring for most of your needs amiright!?
+
+As a start, This mod is based loosley on CSM and will run perfectly on Minecraft 1.12.2 and possibly later versions after that for a cross version mod built entireley from scratch in Minecraft Forge. It brings a number of blocks and items to help make things talk, from NOAA weather radios to the Dectalk, Gravis Ultrasound, Disney Sound Source and many more! If it talks, it will FOR SURE be in this mod!
```

Comment:
```suggestion
This mod is written from the ground up for Forge 1.12 (more versions coming later), using best practices for Forge mod development. It brings a number of blocks and items to help make things talk, from NOAA weather radios to the Dectalk, Gravis Ultrasound, Disney Sound Source and many more! If it talks, it will FOR SURE be in this mod!
```